src/components/TableRow.re

```diff
@@ -0,0 +1,2 @@
+[@react.component]
```

Comment:
would not be better move this to a inside module inside Table component? in practice it does not have as much difference, since this component is only used within Table

just asking

Reply:
Ok, gonna do it